cli/tauri.js/runner.js

```diff
@@ -110,6 +121,34 @@ class Runner {
     }
   }
 
+  __parseHtml(indexPath) {
+    const Inliner = require('inliner')
+    const jsdom = require('jsdom')
+    const { JSDOM } = jsdom
+
+    return new Promise((resolve, reject) => {
+      new Inliner(path.join(indexPath, 'index.html'), (err, html) => {
+        if (err) {
+          reject(err)
+        } else {
+          const dom = new JSDOM(html)
+          const document = dom.window.document
+          document.querySelectorAll('link').forEach(link => {
+            link.removeAttribute('rel')
+            link.removeAttribute('as')
+          })
+
```

Comment:
Brilliant.

Reply:
had to do this, because the webview won't load things with rel=preload